x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -270,17 +270,20 @@ export const EndpointList = () => {
           );
           const toRouteUrl = formatUrl(toRoutePath);
           return (
-            <EndpointListNavLink
-              name={hostname}
-              href={toRouteUrl}
-              route={toRoutePath}
-              dataTestSubj="hostnameCellLink"
-            />
+            <EuiToolTip content={hostname} anchorClassName="eui-fullWidth">
+              <EndpointListNavLink
+                name={hostname}
+                href={toRouteUrl}
+                route={toRoutePath}
+                dataTestSubj="hostnameCellLink"
+              />
+            </EuiToolTip>
           );
         },
       },
       {
         field: 'host_status',
+        width: '9%',
```

Comment:
i'm guessing these hardcoded widths is to make sure the values truncate with ellipsis?

Reply:
it's do restrict some columns that we can predict won't be very long such as the Statuses since they can only have a handful of values.  this frees up room for longer columns like hostname, timestamp, etc